### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by brenoxav**
## Math magicians: components

- Delete all the boilerplate from CRA
- Create a directory called components
- Create a new Calculator.js file
- Create a React component that matches the given design
- Use class based components

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
```
Comment (henatan99) [inline src/App.js:8]: - [ ] This milestone is designed for you to practice React class components. Kindly refactor the `App` component into React class component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <div className="calculator-key">6</div>
+          <div className="calculator-key calculator-key-orange">-</div>
+
+          <div className="calculator-key">1</div>
+          <div className="calculator-key">2</div>
+          <div className="calculator-key">3</div>
+          <div className="calculator-key calculator-key-orange">+</div>
+
+          <div className="calculator-key calculator-key-zero">0</div>
+          <div className="calculator-key">.</div>
```
Comment (henatan99) [inline src/components/Calculator.js:38]: - [ ] These `div` tag elements are supposed to be `button` elements both functionally and semantically. Kindly make changes here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @brenoxav , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (brenoxav) [reply src/App.js:8]: Done.

File: src/components/Calculator.js (hunk 2 above)
Comment (brenoxav) [reply src/components/Calculator.js:38]: Done.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @brenoxav  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**